pushall

```diff
@@ -49,6 +55,18 @@ for DIST in $DISTS; do
     push "$DIST"
 done
 
+if [ -f "build/snapshot_id" ]; then
+    snapshot_id=$(cat build/snapshot_id)
+
+    for DIST in $DISTS_WITH_SNAPSHOT; do
+        DIST="${DIST}-snapshot-${snapshot_id}"
```

Comment:
I would use `TAG` here like in `buildone` so it doesn't shadow the other variable.

Reply:
done